design/scss/custom/_home.scss

```diff
@@ -104,3 +108,13 @@
 		}
 	}
 }
+
+@media (width <= 768px) {
+	.actions {
+		flex-flow: column;
+	}
+
+	.mobile-hide {
+		display: none;
+	}
+}
```

Comment:
please use bootstrap classes instead of custom breakpoints

https://getbootstrap.com/docs/5.3/utilities/flex/
https://getbootstrap.com/docs/5.3/utilities/display/